pfcpiface/fastpath.go

```diff
@@ -9,12 +9,20 @@ import (
 )
 
 type fastPath interface {
+	/* Close any pending sessions */
 	exit()
+	/* setup internal parameters and channel with fastPath */
 	setUpfInfo(u *upf, conf *Conf)
+	/* set udp and pfcp conn parameters in interface */
 	setInfo(udpConn *net.UDPConn, udpAddr net.Addr, pconn *PFCPConn)
+	/* simulator mode setup */
 	sim(u *upf, method string)
+	/* write pdr/far/qer to fastpath */
 	sendMsgToUPF(method string, pdrs []pdr, fars []far) uint8
+	/* delete all pdrs/fars/qers/ installed in fastpath tabled */
 	sendDeleteAllSessionsMsgtoUPF()
+	/* check of communication channel to fastpath is setup */
+	isConnected(accessIP *net.IP) bool
```

Comment:
Doesnt make sense to have accessIP here

Reply:
access IP will be fetched from the p4rt server in case of p4rt fast path. So every time we connect we need to fetch the access IP.